lib/segment/src/fixtures/payload_fixtures.rs

```diff
@@ -117,6 +118,43 @@ pub fn random_vector<R: Rng + ?Sized>(rnd_gen: &mut R, size: usize) -> Vec<Vecto
     (0..size).map(|_| rnd_gen.gen()).collect()
 }
 
+/// Generates a non empty sparse vector
+pub fn random_sparse_vector<R: Rng + ?Sized>(rnd_gen: &mut R, max_size: usize) -> SparseVector {
```

Comment:
`random_sparse_vector` is already presented in `dev` branch

Reply:
good observation :+1: 

decided to unify the fixtures into the sparse crate https://github.com/qdrant/qdrant/pull/2900/commits/3552f08324797a9d7cc71ebc934b3380106aafcb